Assets/MixedRealityToolkit/_Core/Attributes/ExtendsAttribute.cs

```diff
@@ -1,9 +1,11 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using System;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+#if WINDOWS_UWP && !ENABLE_IL2CPP
 using Microsoft.MixedReality.Toolkit.Internal.Extensions;
+#endif // WINDOWS_UWP && !ENABLE_IL2CPP
```

Comment:
What are these for?

Reply:
These were added when I noticed that VS was telling me that the namespace was not being used when building for IL2CPP and removing it blocked building with .NET backend.